client/src/components/to-do-list/ToDoList.jsx

```diff
@@ -2,26 +2,56 @@ import React, {useState, useEffect} from 'react';
 import Categories from './Categories.jsx';
 import Category from './Category.jsx';
 import Tasks from './Tasks.jsx';
+import TestModal from './testModal.jsx';
+import {example} from '../../../.././database/example.js';
 import { Button } from '@material-ui/core';
 import { HTML5Backend } from 'react-dnd-html5-backend';
 import { DndProvider } from 'react-dnd';
 
 const { example } = require('../../../../database/example.js');
 
+
 function ToDoList() {
   //a state prop that's an array that has an element for everytime + task or + category is clicked
-  const [newTasks, setNewTasks] = useState([])
-  const [newCategories, setNewCategories] = useState([])
+  const [categorizedTasks, setCategorizedTasks] = useState([])
+  const [modalOpen, setModalOpen] = useState(false)
+  const [editing, setEditing] = useState(false)
 
-  var deleteTask = (e) => {
+  const deleteTask = (e) => {
     // var target = e.target;
     // console.log(target);
-    e.target.parentNode.style.display = 'none';
-    console.log('deleteTask');
+    // e.target.parentNode.style.display = 'none';
+    console.log('deletetask called');
+  }
+
+  const openModal = () => {
+    console.log('openModal called')
+    setModalOpen(true)
```

Comment:
how do you close the modal? I think I saw this elsewhere but it's best practice to have have the open/close modal to be a single function which is attached to the open modal button, the X, and touch/click off the modal and have that be a single function which toggles.